### PR TITLE
Fix changes in IAM doc v1.7.2

### PR DESCRIPTION
**Pull request by rmiccoli**

### CODE REVIEW

**Comment by jouvin** [comment]
LGTM (as I made the remarks at the origin of this PR). 

There are 2 other oddities that I reported and are not yet addressed by this PR :  

- In JSON web keys page, it would be good to mention that a recent version of Maven, supporting Java 11, is required. On CentOS 8, I used https://linuxize.com/post/how-to-install-apache-maven-on-centos-8/ as a guideline but there are probably others. The whole point is that the Maven version provided by CentOS 8 (and certainly CentOS7 and may be other other distros) works with Java 8 and results in an exception during the build saying that Java 11 is required. 
- At the end of Basic Setup page, it is said to connect to the IAM instance with the default password and change it. The Change Password button is not (no longer?) in the home page but in the "IAM Dashboard" page. Probably updating the image with a more recent version of the app would be good...

File: content/en/docs/getting-started/jwk/_index.md [hunk 1]
```diff
@@ -22,6 +22,8 @@ Build the code with:
 ```shell
 mvn package
 ```
+A Maven version supporting Java 11 is required.
```
Comment (jouvin) [inline content/en/docs/getting-started/jwk/_index.md]: May be give a minimum version number. I have the feeling that it is 3.6.0.

Comment (rmiccoli) [reply content/en/docs/getting-started/jwk/_index.md]: I agree, thanks.

**Comment by jouvin** [comment]
LGTM, would be good to merge it.

Fixes #20.

**Pull request merged**